core/src/main/kotlin/xtdb/api/YamlSerde.kt

```diff
@@ -14,10 +20,59 @@ import xtdb.api.log.InMemoryLogFactory
 import xtdb.api.log.LocalLogFactory
 import xtdb.api.log.LogFactory
 import xtdb.api.storage.ObjectStoreFactory
-import java.util.ServiceLoader
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.*
 import java.util.ServiceLoader.Provider
 import kotlin.reflect.KClass
 
+object EnvironmentVariableProvider {
```

Comment:
(made this into an object because `mockkstatic` was causing failures for SOME reason. Maybe was overriding some of the modules somehow, not sure)